src/www/pages/patterns/header-and-footer-v2/index.njk

```diff
@@ -0,0 +1,590 @@
+{% extends "www/_layouts/layout-left-pane.njk" %}
+{% set pageTitle="Header and footer V2" %}
+{% set section="Patterns" %}
+{# Imports #}
+{% from "www/_partials/component-example/component-example.njk" import compExample %}
+{% from "wmnds/patterns/header-v2/_header-v2.njk" import wmndsHeaderV2 %}
+{% from "wmnds/patterns/footer/_footer.njk" import wmndsFooter %}
+{%- from "wmnds/components/link/link/_link.njk" import wmndsLink -%}
+
+{% block content %}
+{# About #}
+<h2>About</h2>
+{# What #}
+<h3>What does it do?</h3>
+<ul>
+  <li>The primary way of navigating the website</li>
+  <li>Standardised pattern</li>
+</ul>
+{# When #}
+<h3>When to use it?</h3>
+<ul>
+  <li>Every page must have a header and a footer</li>
+</ul>
+<hr>
+<br><br>
+
+
+
+{# Header #}
+<h2>Header</h2>
+{# Skip Link #}
+<h3>Skip link</h3>
+<p>There is a hidden skip link for users with screen reader assistive technology. This allows users to quickly access the main content. All pages must include this.</p>
+<p>Ensure there is a HTML element on the page with the ID of <code class="wmnds-website-inline-code">wmnds-main-content</code> so the user can skip to this section.</p>
+{# Logo #}
+<h3>Logo</h3>
+<p>This must always be the West Midlands Network logo.</p>
+{# Phase indicator #}
+<h3>Phase indicator</h3>
+<p>This component must be used if the service is not live, using the phase indicator guidance.</p>
+{# Navigation #}
+<h3>Navigation</h3>
+<p>The three top-level navigation items were decided by identifying our user’s top tasks using Gerry McGovern’s methodology. We overlaid a heatmap of the home page, where we found the same three areas were the most-used areas.</p>
+<p>The top-level items open a structured megamenu to avoid cognitive overload. Users are familiar with this pattern from online shopping and transport operators.</p>
+<p>We only use three top-level items as we prefer not to use hamburger menus on mobile. Users are required to make an extra click to reveal the navigation, therefore you cannot glance at navigation options.</p>
+{# Mobile apps #}
+<h3>Mobile apps</h3>
+<p>When designing an app, you should use a tab bar and avoid using hamburger items wherever possible. Limit the tabs to the most important tasks using user research.</p>
+{# Search #}
+<h3>Search</h3>
+<p>This functionality only shows for desktop and tablet users. When clicked, search expands and takes over the whole page.</p>
+
+{{
+  compExample([
+    wmndsHeaderV2({
+      navItems: navItems,
+      banner: true,
+      breadcrumbs: {
+        section: section,
+        pageTitle: pageTitle
+      },
+      exampleAria: "Full header"
+    })
+  ])
+}}
+
+{# With mega menu #}
+<h3>With a mega menu</h3>
+<h4>Recommended</h4>
```

Comment:
If the mega menu is the recommended one, should it not be the first example?

Reply:
Yes it should be. Can you remove references to the older header items please @houbly.